### DISCUSSION TITLE
Collapsible Breadcrumbs

### DISCUSSION THREAD

**Discussion by lokeshwarobuli** [mantinedev/mantine#3639] [Feature requests]
Can we please have a collapsible breadcrumbs like in material desgin
References: https://codesandbox.io/s/plwcdx?file=/demo.tsx, https://mui.com/material-ui/react-breadcrumbs/#collapsed-breadcrumbs

![Screenshot 2023-03-01 at 5 59 18 PM](https://user-images.githubusercontent.com/85650351/222285954-621bb802-557f-436b-a434-93fbfadb61bb.png)

**Comment by lokeshwarobuli** [comment]
@rtivital any ideas about this ?

**Reply by rtivital** [reply]
It is in the backlog, I am not planning to work on it soon. You are welcome to submit a PR for this.

**Reply by lokeshwarobuli** [reply]
Thanks

**Comment by lukasbash** [comment]
Would be awesome to have this!
